src/main/java/Baymax.java

```diff
@@ -0,0 +1,61 @@
+import java.nio.channels.ScatteringByteChannel;
+import java.util.*;
+public class Baymax {
+    public static void main(String[] args) {
+        System.out.println("Hello, I am Baymax your personal Chatbot Companion. \nWhat can I do for you today?");
+        makeDecision();
+        System.out.println("See you soon!");
+    }
+
+    public static void makeDecision(){
+        String exit = "bye";
+        String outputL = "list";
+        String mark = "mark";
+        String unmark = "unmark";
+        String to = "todo";
+        String dead = "deadline";
+        String event = "event";
+        String delete = "delete";
+
+        Scanner input = new Scanner(System.in);
+        String currentInput;
+        List<Task> myList = new ArrayList<>();
+        int Counter = 0;
```

Comment:
Variable name doesn't follow coding convention (camel case)
```suggestion
        int counter = 0;
```